recipes/snap/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+make
+mkdir -p $PREFIX/bin
+mkdir -p ${PREFIX}/share/snap
+mkdir -p ${PREFIX}/etc/conda/activate.d ${PREFIX}/etc/conda/deactivate.d
+cp -p snap fathom forge hmm-info exonpairs cds-trainer.pl hmm-assembler.pl noncoding-trainer.pl patch-hmm.pl zff2gff3.pl $PREFIX/bin
+cp -pr HMM ${PREFIX}/share/snap
+cp -pr DNA ${PREFIX}/share/snap
+
+# set environment variables using conda's environment activate/deactivate
```

Comment:
The ZOE environment variable is not set, so no. Any suggestions on how to resolve that? The only other option I can think of for now is replace all binaries with shell script wrappers that set the ZOE environment variable if not set and then call the "real" tool.